pkgs/top-level/all-packages.nix

```diff
@@ -3914,14 +3914,16 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) Security;
   };
 
+  languagetool = callPackage ../tools/text/languagetool {  };
```

Comment:
Why?

Reply:
Please revert that. all-packages is sorted on a best effort basis.